code-studio/src/js/videos.js

```diff
@@ -0,0 +1,312 @@
+'use strict';
+/* global dashboard, Dialog, YT */
+
+var videojs = require('video.js');
+var testImageAccess = require('./url_test');
```

Comment:
This file both moved and renamed from videos.js.erb, with some additional changes necessary.  Major changes:

* Require video.js and url_test.js instead of depending on their global availability.
* Dropped the wrapper function since browserify does that for us now.
* One other adjustment that I'll comment inline.
* Small syntax adjustments to pass linting.